src/components/AddTransaction/index.js

```diff
@@ -0,0 +1,89 @@
+import {useState} from 'react';
+import {useParams} from 'react-router-dom';
+
+import useStore from '../../hooks/useStore';
+import {Form} from '../AddNewAccount/styled';
+
+export default function AddTranscation() {
+	const [transaction, setTransaction] = useState({
+		date: '',
+		change: '',
```

Comment:
Maybe it makes sense to have an `expense: negative number ` , `income: positive number`  🤔
That way your entries stay clean and more specific.